addons/mail/static/src/models/composer_view.js

```diff
@@ -663,6 +644,18 @@ registerModel({
                 }
                 this.composer.thread.unregisterCurrentPartnerIsTyping({ immediateNotify: true });
             }
+            this.update({
+                composition: {
+                    attachments: this.composer.attachments,
+                    id: increment(),
```

Comment:
This is not how `increment` works, it's meant to update the existing value of a field.

Reply:
The counter needs to be stored separately, it can't be the id itself.